### PR TITLE
Push to shared app collection

### PR DESCRIPTION
**Pull request by QuentinBisson**

### CODE REVIEW

**Review by MarcelMue — COMMENTED** [review]
Build is failing because of architect changes or just because flappy?

**Comment by QuentinBisson** [comment]
It appears architect is failing validation

**Comment by ubergesundheit** [comment]
I've prepared https://github.com/giantswarm/architect-orb/pull/251 and https://github.com/giantswarm/architect/pull/600

Trying now

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@2.3.0
+  architect: giantswarm/architect@2.4.0
```
Comment (ubergesundheit) [inline .circleci/config.yml:3]:
```suggestion
  architect: giantswarm/architect@dev:use-architect-with-ct-schema
```

**Comment by ubergesundheit** [comment]
Looks like it works

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@2.3.0
+  architect: giantswarm/architect@dev:use-architect-with-ct-schema
```
Comment (ubergesundheit) [inline .circleci/config.yml:3]: Apply once architect-orb [2.4.1](https://github.com/giantswarm/architect-orb/releases/tag/v2.4.1) has been released

```suggestion
  architect: giantswarm/architect@2.4.1
```

Comment (ubergesundheit) [reply .circleci/config.yml]: Too fast ;)

**Review by ubergesundheit — APPROVED** [review]
LGTM

**Review by rossf7 — APPROVED** [review]

**Pull request merged**